.buildkite/nightly-benchmarks/scripts/run-nightly-benchmarks.sh

```diff
@@ -350,6 +448,9 @@ main() {
   python3 "$BENCHMARK_ROOT/scripts/summary-nightly-results.py"
   upload_to_buildkite
 
+  # run genai-perf tests
```

Comment:
You can try to uncomment this, then add `perf-benchmarks` label into the PR to try running it on the Perf Benchmark pipeline on Buildkite.

Reply:
@jikunshang Just to confirm your plan—are you planning to add placeholder code and update the pip dependency in this PR, then make it functional in the next PR? If so, we encourage merging these two PRs into one and ensuring it is functional. We're hesitant to include placeholder code alone in the vLLM codebase.

Reply:
I think merge into one PR is better.  let me enable this.